src/components/Tags/__tests___/Tag.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import { colors, radius, sizes, typography } from '@magnetis/astro-tokens';
+
+import Tag from '../Tag';
+import type { TagProps } from '../Tag';
+import type { FontWeightType } from '@typings/styles';
+
+const initialProps: TagProps = {
+  variant: 'neutral',
+  type: 'default',
+  text: 'some tag',
+  testID: 'Tag',
+};
+
+describe('Tag', () => {
+  it('renders correctly with default props', () => {
+    const { getByTestId, getByText } = render(<Tag {...initialProps} />);
+
+    const tag = getByTestId('Tag');
+    const tagLabel = getByText('some tag');
+
+    expect(tag).toHaveStyle({
+      backgroundColor: colors.solidBrightLight,
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      minWidth: sizes.great,
+      height: sizes.smaller,
+      borderRadius: radius.mini,
+      paddingVertical: sizes.quark,
+      paddingHorizontal: sizes.nano,
+    });
+
+    expect(tagLabel).toHaveStyle({
+      color: colors.solidFaintLight,
+      fontSize: typography.fontSizeMini,
+      fontFamily: typography.fontFamilySupport,
+      fontWeight: typography.fontWeightMedium.toString() as FontWeightType,
```

Comment:
❓ Does the following wokrs?
```suggestion
      fontWeight: `${typography.fontWeightMedium}`,
```

Reply:
Yes, it does! Changed in [500aaba](https://github.com/magnetis/astro-native/pull/791/commits/500aabac63e9b2e74ed6332f3c9727f6edc33224)